tech.json

```diff
@@ -1479,7 +1488,7 @@
         },
         {
           "name": "canXMode",
-          "requires": ["XRayScope"],
+          "requires": ["XRayScope", {"noFlashSuit": {}}],
```

Comment:
One of the ways to activate canSuperJump is with a FlashSuit in XMode.

Reply:
Ah, right. I think it might still be good to have the "noFlashSuit" here, since any flash suit will be lost after releasing X-Ray (when X-mode is canceled). That kind of super jump strat could be represented by having a "useFlashSuit" requirement before the "canXMode" requirement; since the flash suit usage happens during the X-mode, it doesn't seem any more inaccurate to put it before "canXMode" rather than after? We could also make a helper for that combination to avoid risk of mixing it up?

Reply:
A helper would be nice.  I mostly need a note somewhere since I'll forget.

Reply:
Done.